Library/Homebrew/rubocops/mixins/on_formula_class.rb

```diff
@@ -0,0 +1,18 @@
+module RuboCop
+  module Cop
+    # Helper module for checking formula class
+    module FormulaDef
```

Comment:
Yeah, this makes more sense, but I was not able to inherit `HomebrewCop` due to some error. Found a fix. Will push the changes.